evap/evaluation/migrations/0057_auto_20170619_2238.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-06-19 22:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.utils import timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0056_alter_userprofile_email'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='course',
+            name='vote_start_date',
+        ),
+        migrations.AddField(
+            model_name='course',
+            name='vote_start_datetime',
+            field=models.DateTimeField(default=timezone.now(), verbose_name='start of evaluation'),
+            preserve_default=False,
```

Comment:
this will discard all the data as far as i can see. i think you actually need to have an AlterField and RenameField.